.github/workflows/publish.yml

```diff
@@ -44,3 +44,29 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  website:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: install pnpm
+        uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 7.4.0
+
+      - name: setup node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'pnpm'
+
+      - name: install dependencies
+        run: pnpm install
+
+      - name: deploy website
+        run: pnpm run -r deploy
```

Comment:
Might want to combine this into the main job, e.g. via changeset's command for releasing a new version.

The docs should be versioned, and I would expect documentation for new versions to go out at the same time as the packages. 

Conversely, coupling this to the `packages` job could also make it harder to re-attempt deploying the docs when all other steps are completed successfully.